crates/containerd-shim-wasm/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 - Improved the getting started guide and Makefile for Windows ([#665](https://github.com/containerd/runwasi/pull/665))
 - Modified behavior so that container environment variables are exclusively passed to WASI modules, enhancing security and isolation. ([#668](https://github.com/containerd/runwasi/pull/668))
 - Updated the `containerd-shim` dependency to the latest version.
+- Update the `opentelemetry` related dependencies to the latest version ([#712](https://github.com/containerd/runwasi/pull/712))
```

Comment:
Please add this to the unreleased section

Reply:
Sorry, I hadn't noticed the format. Done :-)